src/fs_storage.rs

```diff
@@ -478,6 +496,12 @@ mod tests {
         tests::test_storage_ext_compact(storage);
     }
 
+    #[test]
+    fn test_applied() {
+        let (_tmp, storage) = create_temp_storage("test_storage_ext_compact");
```

Comment:
copy-pasta: `create_temp_storage("test_storage_ext_compact")` should be `create_temp_storage("test_applied")`